www/README.md

```diff
@@ -42,13 +42,13 @@ ANALYTICS_SERVICE_ACCOUNT_KEY="PEM KEY VALUE"
 
 ### Enabling localizations
 
-Localizations are currently a work-in-progress and are thus disabled by default. They can be enabled by setting the `ENABLE_LOCALIZATIONS` env variable:
+Localizations are currently a work-in-progress and are thus disabled by default. They can be enabled by setting the `LOCALES` env variable to the locales you want to build:
```

Comment:
I'd link to the json file here for reference 👍 
At least I don't know all language codes by hard 😆

Reply:
what, you don't have the full list of ISO 639-1 language codes memorized? :P